tokens/style-dictionary.js

```diff
@@ -154,4 +154,24 @@ StyleDictionary.registerFormat({
   },
 });
 
+/**
+ * Formatter to generate CSS custom media queries.
+ * Gets input about existing tokens of the 'size' category,
+ * 'breakpoints' subcategory, and generates a CSS custom media queries.
+ */
+StyleDictionary.registerFormat({
+  name: 'css/custom-media',
+  formatter({ dictionary, file }) {
+    const { size: { breakpoint } } = dictionary.properties;
```

Comment:
This is neat to see that this wasn't that large of a lift given what we already had! 🚀

Reply:
Yes, it's very simple thanks to the Style Dictionary setups 💯